simulation/simulation_interface/proto/geometry_msgs.proto

```diff
@@ -1,58 +1,65 @@
 syntax = "proto3";
 
 /**
- * Protobuf package to define geometry_msgs in ROS2.(https://github.com/ros2/common_interfaces/tree/master/geometry_msgs)
-*/
+ * Protobuf package to define geometry_msgs in
+ * ROS2.(https://github.com/ros2/common_interfaces/tree/master/geometry_msgs)
+ */
 package geometry_msgs;
 
 /**
- * Protobuf definition of [geometry_msgs/msg/Point type in ROS2.](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Point.msg)
-**/
+ * Protobuf definition of [geometry_msgs/msg/Point type in
+ *ROS2.](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Point.msg)
+ **/
 message Point {
-    double x = 1; // X value in a cartesian coordinate.
-    double y = 2; // Y value in a cartesian coordinate.
-    double z = 3; // Z value in a cartesian coordinate.
+  double x = 1; // X value in a cartesian coordinate.
+  double y = 2; // Y value in a cartesian coordinate.
+  double z = 3; // Z value in a cartesian coordinate.
 }
 
 /**
- * Protobuf definition of [geometry_msgs/msg/Quaternion type in ROS2.](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Quaternion.msg)
-**/
+ * Protobuf definition of [geometry_msgs/msg/Quaternion type in
+ *ROS2.](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Quaternion.msg)
+ **/
```

Comment:
Malformed indentation

Reply:
fixed in fixed in https://github.com/tier4/scenario_simulator_v2/pull/357/commits/082e52f817ed4a829eeddfdd9148cd0455d8661b